specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/examples/ClusterExtensionTypes_List.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters": {
+    "subscriptionId": "aaaaaaaaaa",
+    "resourceGroupName": "rgextensionTypes",
+    "clusterRp": "Microsoft.ContainerService",
+    "clusterResourceName": "managedClusters",
+    "clusterName": "aaaaaaaaaaaaaaaaaaaaa",
+    "api-version": "2022-01-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "releaseTrains": [
+                "aaaaaaaaaaaaaaaaaaaaaa"
+              ],
+              "clusterTypes": "connectedClusters",
+              "supportedScopes": {
+                "defaultScope": "aaaaaaaaaaa",
```

Comment:
Could you provide more meaningful value for those properties?

Reply:
Sure, makes sense. Will update these